src/include/common/timer.h

```diff
@@ -43,16 +43,33 @@ class Timer : public peloton::Printable {
             .count();
 
     elapsed_ += duration;
+    total_ += duration;
     invocations_++;
   }
 
-  inline void Reset() { elapsed_ = 0; }
-
-  // Get Elapsed duration
-  inline double GetDuration() const { return elapsed_; }
-
-  // Get Number of invocations
-  inline int GetInvocations() const { return invocations_; }
+  /**
+   * Reset the current elapsed time counter.
+   */
+  void Reset() { elapsed_ = 0; }
+
+  /**
+   * Get elapsed duration
+   * @return
+   */
+  double GetDuration() const { return elapsed_; }
+
+  /**
+   * Return the total amount of time measured by
+   * this timer after multiple invocations.
+   * @return
+   */
+  double GetTotalDuration() const { return (total_); }
+
+  /**
+   * Get number of invocations
+   * @return
```

Comment:
It looks like the @return tag is not filled out. (I personally leave out the general comment and just write the return tag for simple functions like this, i.e. @return number of invocations)